lib/types/SSITypesBuilder.ts

```diff
@@ -53,22 +61,29 @@ export class SSITypesBuilder {
   }
 
   static mapExternalVerifiablePresentationToWrappedVP(
-    presentation: IPresentation | JwtDecodedVerifiablePresentation | string
+    presentation: OriginalVerifiablePresentation | IPresentation
   ): WrappedVerifiablePresentation {
     const isJwtEncoded: boolean = ObjectUtils.isString(presentation);
-    const type: OriginalType = isJwtEncoded ? OriginalType.JWT_ENCODED : OriginalType.JSONLD;
-    let vp = isJwtEncoded ? this.decodeJwtVerifiablePresentation(presentation as string) : presentation;
-    vp = this.isJwtDecodedPresentation(vp)
+    const type: OriginalType = isJwtEncoded
+      ? OriginalType.JWT_ENCODED
+      : this.isJwtDecodedPresentation(presentation)
+      ? OriginalType.JWT_DECODED
+      : OriginalType.JSONLD;
+    let vp = isJwtEncoded ? this.decodeJwtVerifiablePresentation(presentation as CompactJWT) : presentation;
+    vp = this.isJwtDecodedPresentation(vp as OriginalVerifiablePresentation)
       ? this.createInternalPresentationFromJwtDecoded(vp as JwtDecodedVerifiablePresentation)
       : vp;
     const vcs: WrappedVerifiableCredential[] = this.mapExternalVerifiableCredentialsToWrappedVcs(
       (vp as IPresentation).verifiableCredential
     );
+    const format: PresentationFormat =
+      type === OriginalType.JWT_ENCODED || type === OriginalType.JWT_DECODED ? 'jwt' : 'ldp';
```

Comment:
```suggestion
      type === OriginalType.JWT_ENCODED || type === OriginalType.JWT_DECODED ? 'jwt_vp' : 'ldp_vp';
```

If we know the type use the full type. ldp and jwt typically need context on the processing side. We can provide a more explicit type as we know what we are creating, so use that.